main.go

```diff
@@ -22,18 +22,19 @@ import (
 	"fmt"
 	"os"
 	"runtime"
-	. "github.com/uniqush/uniqush-push/srv"
+
+	"github.com/uniqush/uniqush-push/srv"
 )
 
 var uniqushPushConfFlags = flag.String("config", "/etc/uniqush/uniqush-push.conf", "Config file path")
 var uniqushPushShowVersionFlag = flag.Bool("version", false, "Version info")
 
-var uniqushPushVersion = "uniqush-push 1.5.2"
+var uniqushPushVersion = "uniqush-push 1.6.0"
```

Comment:
Since it's a major change that could break old clients expecting the old style of reply, I'd personally opt for going up a major version to 2.0.0